ImputationPipeline/notebooks/CreateSampleSets.ipynb

```diff
@@ -28,77 +28,97 @@
     "workspace_name = os.environ.get('WORKSPACE_NAME')\n",
     "\n",
     "\n",
-    "# Download and unzip current sample_set table\n",
-    "print('Downloading and unzipping sample_set table...')\n",
-    "sample_set_response = fapi.get_entities_tsv(workspace_namespace, workspace_name, 'sample_set')\n",
-    "if not sample_set_response.ok:\n",
-    "    raise RuntimeError(f'ERROR: {sample_set_response.text}')\n",
+    "def group_samples_into_batches(table_name, available_tables):\n",
```

Comment:
There are some changes here, but the comparison thinks it is much more than it really is because it's just getting confused by an extra tab as everything moves inside a function.